sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/AzureMonitorHelper.java

```diff
@@ -39,8 +41,7 @@ public static TelemetryItemExporter createTelemetryItemExporter(HttpPipeline htt
                         50, // default to 50MB
                         TempDirs.getSubDir(tempDir, "telemetry"),
                         telemetryPipeline,
-                        // TODO (trask) change this to statsbeatModule.getNonessentialStatsbeat()?
```

Comment:
exporter public API needs to be updated for customers to turn non-essential on. until then, default is OFF.

Reply:
can you move this TODO up to the caller now? I suspect we will enable non-essential statsbeat by default to align with distro (and we can delete the TODO if we do not)